app/src/main/java/grinnell/appdev/edu/lyles/adapter/ItemAdapter.java

```diff
@@ -131,33 +129,34 @@ else if (mExpandedIndex == itemPosition) {
                             mExpandedIndex = NONE_SELECTED;
                         }
                     }
-                    mMenuItems.remove(clickedItem);
+                    mLylesMenuItems.remove(clickedItem);
                     notifyItemRemoved(itemPosition);
                 }
             }
         });
 
         TextView detailsTextView = viewHolder.mDetailsTextView;
-        detailsTextView.setText(menuItem.getDetails());
+        detailsTextView.setText(lylesMenuItem.getDetails());
     }
 
     @Override
     public int getItemCount() {
-        return mMenuItems.size();
+        return mLylesMenuItems.size();
     }
 
     private Context getContext() {
         return mContext;
     }
 
     /**
-     * Changes a view to or from its expanded mode
+     * Changes a view to or from its expanded mode to its contracted mode
```

Comment:
Slightly confusing wording. Also, if this toggles/switches between contracted and expanded modes the method name should say `toggle` rather than just `expand`.